server/mediaserver.go

```diff
@@ -72,8 +72,15 @@ func NewLivepeerServer(rtmpPort string, httpPort string, ffmpegPath string, lpNo
 func (s *LivepeerServer) StartMediaServer(ctx context.Context, maxPricePerSegment int, transcodingOptions string) error {
 	if s.LivepeerNode.Eth != nil {
 		BroadcastPrice = big.NewInt(int64(maxPricePerSegment))
-		BroadcastJobVideoProfile = types.VideoProfileLookup[transcodingOptions]
-		glog.Infof("Transcode Job Price: %v, Transcode Job Type: %v", BroadcastPrice, BroadcastJobVideoProfile)
+		bProfiles := make([]types.VideoProfile, 0)
+		for _, opt := range strings.Split(transcodingOptions, ",") {
```

Comment:
Should we also trim whitespace, in case the opts were passed in as comma separated with a space?

Reply:
Good point